tpp-app/src/home/pages/SelectLogOptionOverlay.js

```diff
@@ -5,6 +5,7 @@ import { useBottomTabBarHeight } from '@react-navigation/bottom-tabs';
 import { TabBarMiddleButton } from '../components/TabBarMiddleButton'
 import BloodDrop from '../../../ios/tppapp/Images.xcassets/icons/blood_drop';
 import Calendar from '../../../ios/tppapp/Images.xcassets/icons/calendar_icon_multiple_dates';
+import { STACK_SCREENS } from '../CalendarNavigator';
```

Comment:
We updated to CALENDAR_STACK_SCREENS for this one, so you'll have to change this for it to run.